sdk/python/feast/templates/local/feature_store.yaml

```diff
@@ -1,5 +1,6 @@
 project: my_project
-registry: data/registry.db
+registry:
+    path: data
```

Comment:
maybe this should be "data/" to make it clear it's a dir. Unsure tho

Reply:
Good point, I think it works either way in the code, but for the template and the docs it could be better to have trailing `/`